### PR TITLE
[tests/acceptance] Add OS requirements file and rewrite README

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

**Comment by lluiscampos** [comment]
Follow-up from latest Mender QA Architecture meeting.

See also https://github.com/mendersoftware/mender-test-containers/pull/57

File: tests/acceptance/README.md [hunk 1]
```diff
@@ -5,34 +5,11 @@ Acceptance tests for Mender
 Prerequisites
 -------------
 
-* PyTest
-* [Fabric](http://www.fabfile.org)
+* OS dependencies, assuming Debian based, at `requirements_deb.txt`.
```
Comment (oleorhagen) [inline tests/acceptance/README.md:8]: `s/at/in/`

File: tests/acceptance/requirements_deb.txt [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+qemu-utils
+s3cmd
+screen
+sudo
+sysbench
+sysstat
+texinfo
+wget
+zlib1g-dev
+rustc
```
Comment (oleorhagen) [inline tests/acceptance/requirements_deb.txt:58]: Unrelated to this PR, but I don't fully understand why the rust compiler is needed, like it seems to me that prebuilt binaries are available for `x86` [here](https://pypi.org/project/cryptography/#files)

**Review by oleorhagen — APPROVED** [review]
Nice unification

File: tests/acceptance/requirements_deb.txt (hunk 2 above)
Comment (mzedel) [reply tests/acceptance/requirements_deb.txt:58]: you might find [this](https://github.com/mendersoftware/mender-test-containers/pull/49/files/9930baa667b772e088a2d29ddcce6b41f7ec369e#diff-10213bb36066710156ec0a50127d6bd0e8117030ea1a6bc2357ae0d5e55b8e31) relevant - TLDR: ubuntu package updates are just too damn slow for us 😉

File: tests/acceptance/README.md [hunk 3]
```diff
@@ -5,34 +5,11 @@ Acceptance tests for Mender
...
-you run them again a second time without rebuilding the image first.
-
-
-Troubleshooting
----------------
-
-A common problem is that the SSH server inside the image stops working. This
-often happens if you run the tests several times after one another. It is
-unknown why, but rebuilding the Yocto image seems to make the problem go away.
+See https://github.com/mendersoftware/mender-qa/blob/master/Documentation/test-suites-overview.md#client-acceptance-tests
```
Comment (mzedel) [inline tests/acceptance/README.md:15]: 👍 I like the centralization - less places to maintain 🙂

**Review by mzedel — APPROVED** [review]

File: tests/acceptance/requirements_deb.txt (hunk 2 above)
Comment (oleorhagen) [reply tests/acceptance/requirements_deb.txt:58]: That explains it!
I find that I'm not missing Ubuntu much

Comment (lluiscampos) [reply tests/acceptance/requirements_deb.txt:58]: Soon (in theory this month) the next yocto release will be out, then we migrate, then we deprecate `warrior`, and *then* we update the base image to next Ubuntu LTS. Be patient my friends :)

File: tests/acceptance/README.md (hunk 3 above)
Comment (lluiscampos) [reply tests/acceptance/README.md:15]: Having the doc here and a pointer in `mender-qa` is also okey with me (that is the case for integ tests, for example). What is not okey is to copy it.

I guess I still have in my veins C coding principles :P

**Comment by lluiscampos** [comment]
The CI checks will not arrive, as GitLab is down ATM. Last push only modified `README`, so this is good to go. Merging.

File: tests/acceptance/requirements_deb.txt (hunk 2 above)
Comment (kacf) [reply tests/acceptance/requirements_deb.txt:58]: Sorry to burst the bubble, but we only follow LTS releases now, and the next one is not out until April 2022. :-)

Dunfell will live on for one more year! And warrior, well maybe also one year, but it is not LTS, so we might want to discuss that.

**Pull request merged**